Dockerfile.gykernel

```diff
@@ -1,56 +1,30 @@
 FROM ubuntu:16.04
 
 RUN apt-get update && apt-get install -y \
-        build-essential \
-        curl \
-        git \
-        libfreetype6-dev \
-        libpng12-dev \
-        libzmq3-dev \
-        mlocate \
-        pkg-config \
-        python-dev \
-        python-numpy \
-        python-pip \
-        software-properties-common \
-        swig \
-        zip \
-        zlib1g-dev \
-        libcurl3-dev \
-        openjdk-8-jdk\
-        openjdk-8-jre-headless \
-        wget
+	curl \
+	python3-pip
 
-# dependencie for tensorflow serving
-RUN apt-get install libstdc++6 && \
-    add-apt-repository ppa:ubuntu-toolchain-r/test && \ 
-    apt-get update && \
-    apt-get -y upgrade && \
-    apt-get -y dist-upgrade
+RUN pip3 install --upgrade tensorflow-serving-api-python3
 
-# clean up
-RUN apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN echo "deb [arch=amd64] \
+http://storage.googleapis.com/tensorflow-serving-apt \
+stable tensorflow-model-server tensorflow-model-server-universal" \
+| tee /etc/apt/sources.list.d/tensorflow-serving.list
 
+RUN curl \
+https://storage.googleapis.com/tensorflow-serving-apt/tensorflow-serving.release.pub.gpg \
+| apt-key add -
 
-# install tensorflow
-RUN pip install tensorflow
-
-# install tensorflow serving
-RUN pip install tensorflow-serving-api
-RUN echo "deb [arch=amd64] http://storage.googleapis.com/tensorflow-serving-apt stable tensorflow-model-server tensorflow-model-server-universal" | \
-        tee /etc/apt/sources.list.d/tensorflow-serving.list
-RUN curl https://storage.googleapis.com/tensorflow-serving-apt/tensorflow-serving.release.pub.gpg | apt-key add -
-RUN apt-get update && apt-get install -y tensorflow-model-server
-
-# Set up grpc
-
-RUN pip install enum34 futures mock six && \
-    pip install --pre 'protobuf>=3.0.0a3' && \
-    pip install -i https://testpypi.python.org/simple --pre grpcio
-
-
-RUN cd /root && git clone https://github.com/BiondiVini/gykernel.git
-
-ENTRYPOINT ["tensorflow_model_server"]
-CMD ["--port=7000", "--model_name=inception", "--model_base_path=/root/gykernel/saved_models"]
+RUN apt-get update && apt-get install -y \
+	tensorflow-model-server && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/*
+	
+EXPOSE 7000
+
+ADD saved_models /models
+
+ENTRYPOINT [ "tensorflow_model_server", \
+"--model_base_path=/model", \
```

Comment:
just to remember, this parameters model_base_path and model_name should be replaced by a configuration file